config/locales/en.yml

```diff
@@ -333,6 +333,42 @@ en:
               <li>who will be interviewing them</li>
               <li>any tests needed - if so, how they can prepare</li>
             </ul>
+      school_placements:
+        edit:
+          how_school_placements_work: How school placements work
+          page_title: How school placements work - %{course_name_and_code}
+          submit_button: Update how school placements work
+          guidance_text_html:
+            <p class="govuk-body">Give candidates information about the schools they will be training in</p>
+            <p class="govuk-body">Tell them:</p>
+            <ul class="govuk-list govuk-list--bullet">
+              <li>how many placements they will have</li>
+              <li>how much time they will spend in each school</li>
+              <li>if mentors are available within the schools</li>
+            </ul>
```

Comment:
I've never seen markup in locale files

Reply:
We have them in Register too. All the footer pages used to be in locales so content designers could update them themselves, but we moved away from it a while ago. 

I prefer these being in the HTML file unless we reuse the content a lot but hey ho